dev/com.ibm.ws.jaxb_fat/publish/servers/com.ibm.ws.jaxb.ContextServer/server.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+    Copyright (c) 2018 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+    <featureManager>
+        <feature>servlet-3.1</feature>
+        <feature>componenttest-1.0</feature>
+        <feature>jaxb-2.2</feature>
```

Comment:
If only going to be used for EE10 might as well make the feature list an EE10 feature list.